import_db.py

```diff
@@ -175,6 +175,7 @@ def main():
     if args.import_home:
         username = args.import_home
         print("Importing home directory: {0}".format(username))
+        client_config['overwrite'] = args.overwrite
```

Comment:
We should not add the overwrite flag here, it should be added to `build_client_config()` in the parser.py file. If the arg exists, we add it to the client configs. This would allow us to re-use this flag for `--import-home` or general workspace imports.